specification/ai/Azure.AI.Projects/red-teams/models.tsp

```diff
@@ -0,0 +1,118 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@typespec/openapi";
+import "../common/models.tsp";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Projects;
+
+@doc("Strategies for attacks.")
+union AttackStrategy {
+    string,
+
+    @doc("Represents an easy attack strategy.")
```

Comment:
Represents a default set of easy complexity attacks. Easy complexity attack strategies are defined as attacks that do not require any Large Language Model to convert or orchestrate.